lib/matplotlib/tests/test_image.py

```diff
@@ -895,7 +895,9 @@ def test_imshow_float128():
     fig, ax = plt.subplots()
     ax.imshow(np.zeros((3, 3), dtype=np.longdouble))
     # Ensure that drawing doesn't cause crash
-    fig.canvas.draw()
+    # & catch a warning that float128 gets cast to float64
+    with pytest.warns(UserWarning):
```

Comment:
This one fails the Appveyor build with no Warning...

Reply:
Hmm, must be a platform specific warning.